Libraries/LibIDL/IDLParser.cpp

```diff
@@ -51,6 +52,26 @@
     exit(EXIT_FAILURE);
 }
 
+static bool is_valid_cpp_identifier_start(char start)
+{
+
+    // https://en.cppreference.com/w/cpp/language/identifiers
+    // to be a valid C++ identifier it must be a:
+
+    // uppercase Latin letters A-Z
+    // lowercase Latin letters a-z
+    if (isalpha(start)) {
```

Comment:
`is_ascii_alpha(start)` can be used here, which avoids needing to include `ctype.h`.